lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -42,7 +42,7 @@ class OffscreenImages extends Audit {
       informative: true,
       helpText: 'Images that are not above the fold should be lazily loaded after the page is ' +
         'interactive. Consider using the [IntersectionObserver](https://developers.google.com/web/updates/2016/04/intersectionobserver) API.',
-      requiredArtifacts: ['ImageUsage', 'ViewportDimensions', 'traces', 'networkRecords']
+      requiredArtifacts: ['ImageUsage', 'ViewportDimensions', 'traces', 'devtoolsLogs']
```

Comment:
oh, I see. `const Audit = require('./byte-efficiency-audit');` makes that difficult to spot. We should probably rename at some point :)